leetcode/src/2331. Evaluate Boolean Binary Tree.cpp

```diff
@@ -0,0 +1,10 @@
+class Solution {
```

Comment:
This is not `C` - the `C` language doesn't even have the keyword `class` in it. What you have is a solution to the problem in `C++` and whether the source code is written inside the `.c` file or `.cpp` file has nothing to do with the language that is used.